bears/ruby/RubySmellBear.py

```diff
@@ -162,7 +162,7 @@ def generate_config(self,
             "UncommunicativeModuleName": bad_module_name,
             "UncommunicativeParameterName": bad_param_name,
             "UncommunicativeVariableName": bad_var_name,
-            "UnusedParameters": check_unused_variables,
+            "UnusedParameters": not allow_unused_variables,
             "UnusedPrivateMethod": check_unused_private_methods,
```

Comment:
Hm aren't we changing for `check_unused_private_methods` ? In such case it should go in separate commit.
Btw the commit message isn't appropriate. It should rather be 
```
Changes ``check_unused_variables``->
``allow_unused_variables``.
```